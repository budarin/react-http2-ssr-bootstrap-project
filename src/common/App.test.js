const timeout = 10000;

describe(
    '/ (Home Page)',
    () => {
        let page;
        beforeAll(async () => {
            page = await global.browser.newPage();
            await page.goto('https://localhost');
        }, timeout);

        it('should load without error', async () => {
            const text = await page.evaluate(() => document.body.textContent);
            expect(text).toContain('Hello World');
        });
    },
    timeout,
);
