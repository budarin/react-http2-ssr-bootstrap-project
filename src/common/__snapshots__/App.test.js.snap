// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page renders correctly 1`] = `
<p>
  Hello World!
</p>
`;
